docs/modules/data-structures/pages/updating-map-entries.adoc

```diff
@@ -408,8 +411,9 @@ Go::
 [source,go]
 ----
 // error checking is omitted for brevity
-studentMap, _ := hzClient.GetMap("studentMap")
-studentMap.RemoveAll(predicate.Equal("GradYear", 2020))
+ctx := context.Background()
```

Comment:
Even the Java sample is not correct for this section. How about fixing them all?

Reply:
I think let's fix the Go bits and notify the documentation team about code for other clients.